utils/tests/test_hosts.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+""" Tests utils/hosts.py """
+import re
+import pytest
+import os
+from unittest import mock
+import logging
+
+from locust.env import Environment
+
+from utils.hosts import MilMoveHostMixin, MilMoveDomain, MilMoveEnv, clean_milmove_host_users
+from utils.base import ImplementationError
+
+
+class TestMilMoveDomain:
+    """ Tests the MilMoveDomain class and its methods. """
+
+    def test_host_name(self):
+        assert (
+            MilMoveDomain.match(MilMoveDomain.PRIME).host_name("local", True, 1111, "http") == "http://primelocal:1111"
+        )
+        assert MilMoveDomain.match(MilMoveDomain.PRIME).host_name("exp") == "https://prime.exp.move.mil"
+
+        with pytest.raises(ImplementationError):
+            MilMoveDomain.match(MilMoveDomain.PRIME).host_name("local", True, 11111)
+
+        with pytest.raises(ImplementationError):
+            MilMoveDomain.match(MilMoveDomain.PRIME).host_name("test")
+
+
+class TestMilMoveHostMixin:
+    """ Tests the MilMoveHostMixin class and its methods. """
+
+    @classmethod
+    def setup_class(cls):
+        """ Define and initialize classes to be tested """
+
+        class HostLocal(MilMoveHostMixin):
+            # These attributes are used in MilMoveHostMixin to set up the proper hostname for any MilMove environment:
+            local_port = "9443"
+            domain = MilMoveDomain.PRIME  # the base domain for the host
+            is_api = True  # if True, uses the api base domain in deployed environments
+            host = "local"
+
+        """ Initialize the APIParser that will be tested. """
```

Comment:
In general, it's good to avoid using docstring comments unless they are documenting a class or a function definition. The reason for this is because they aren't ignored by the compiler like normal comments, so it creates unnecessary noise when you run the code.